src/jaxsim/high_level/model.py

```diff
@@ -949,6 +949,19 @@ def com_position(self) -> jtp.Vector:
     # Algorithms
     # ==========
 
+    @functools.partial(oop.jax_tf.method_ro)
+    def coriolis_matrix(self) -> jtp.Matrix:
+        from jaxsim.physics.algos.coriolis import coriolis
+
+        H, H_dot, C = jaxsim.physics.algos.coriolis.coriolis(
+            model=self.physics_model,
+            q=self.data.model_state.joint_positions,
+            qd=self.data.model_state.joint_velocities,
+            xfb=self.data.model_state.xfb(),
+        )
+
+        return H, H_dot, C
```

Comment:
Also, similarly to the other high-level methods, the returned matrices should be converted to the active velocity representation.

Reply:
I have some notes on this in https://github.com/ami-iit/idynfor/blob/master/doc/theory_background.md#dynamics , even if the do not covert the 
$\dot{M}$ and $C$ case at the moment.